mmv1/third_party/terraform/tests/resource_cloudbuild_worker_pool_test.go.erb

```diff
@@ -151,6 +151,7 @@ resource "google_cloudbuild_worker_pool" "pool" {
 	}
 	network_config {
 		peered_network = google_compute_network.network.id
+		peered_network_ip_range = "/25"
```

Comment:
This field isn't actually added to the resource yet!

Reply:
Hi Riley,
Thanks for looking at my CL - I ran the 'make upgrade-dcl' script to add my DCL changes, which fixed this test. However, my changes were not related to the other tests which are failing - how should I proceed?

Reply:
The remaining failure- TestAccCloudfunctions2function_cloudfunctions2BasicGcsExample- is failing for an unrelated reason.